region/maridia/inner-pink/Halfie Climb Room.json

```diff
@@ -975,6 +1004,35 @@
         }
       }
     },
+    {
+      "link": [4, 4],
+      "name": "Leave With Runway - Frozen Mochtroid",
+      "requires": [
+        "h_canNavigateUnderwater",
+        "h_canFrozenEnemyRunway",
+        {"or": [
+          "canMochtroidIceClimb",
```

Comment:
Little bit obnoxious to set up.  I find it more likely to take damage too, especially when positioning the runway freeze.  It is also a bit of a choice to stand with Mochtroid attached at the correct height to freeze vs timing it as it floats up somewhat fast.